rpc/README.md

```diff
@@ -6566,6 +6578,45 @@ The information about an active running protocol.
 
 * `version`: `string` - Active protocol version.
 
+### Type `ResponseFormat<BlockView>`
+This is a wrapper for JSON serialization to select the format between Json and Hex.
+
+###### Examples
+
+`ResponseFormat<BlockView>` returns the block in its Json format or molecule serialized Hex format.
+
+#### Fields
+
+`ResponseFormat<BlockView>` is a JSON object with the following fields.
+
+* `inner`: [`Either<BlockView | JsonBytes>`](#type-either_for_blockview_and_jsonbytes) - The inner value.
+
+### Type `ResponseFormat<HeaderView>`
+This is a wrapper for JSON serialization to select the format between Json and Hex.
+
+###### Examples
+
+`ResponseFormat<BlockView>` returns the block in its Json format or molecule serialized Hex format.
+
+#### Fields
+
+`ResponseFormat<HeaderView>` is a JSON object with the following fields.
+
+* `inner`: [`Either<HeaderView | JsonBytes>`](#type-either_for_headerview_and_jsonbytes) - The inner value.
```

Comment:
The type `Either` has not been generated, but there is a link.

Reply:
yes, because this is a hack fix for schemars, schemars respect this option:
https://github.com/nervosnetwork/ckb/blob/ed1c72fae1ef50be982158ab7764cdea02de13bc/util/jsonrpc-types/src/lib.rs#L87

but we don't know how to determine whether to show the link, so it's safe to show all links for types(even in small possibility the link is only works as a dead link as `#`).

similar scenarios happened in the previous version of doc generator.